platzky/shortcodes/shortcode.py

```
@@ -0,0 +1,172 @@
+
+        Returns:
+            Parsed value, or the declared default, or empty string.
+
+        Raises:
+            AttributeError: If name is not in the schema.
+        """
+        if name in self._values:
+            return self._values[name]
+        if name in self._schema:
+            return self._schema[name].default or ""
+        raise AttributeError(f"No shortcode attribute {name!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)